src/lib/auth/authconf.rb

```diff
@@ -755,18 +755,99 @@ def aux_apply
                 service_disable_stop('nscd')
             end
         end
-        
+
+        def IsInstalledVersionNewerOrEqual(installed_rpm_version, test_rpm_version)
```

Comment:
Please, use snake case for method names:
```suggestion
        def is_installed_version_newer_or_equal(installed_rpm_version, test_rpm_version)
```